src/seedu/addressbook/commands/EditPhoneCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.Phone;
+
+public class EditPhoneCommand extends Command {
+    public static final String COMMAND_WORD = "edit_phone";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates the phone number of a user.\n"
+            + "Parameters: Old Phone n/New Phone\n"
+            + "Example: " + COMMAND_WORD
+            + " 88888888 n/81818181";
+    private static final String MESSAGE_DUPLICATE = "Cannot update address book to same phone number";
+    public static final String MESSAGE_SUCCESS = "Person updated: %1$s";
+    private String oldPhone, newPhone;
+    public EditPhoneCommand(String oldPhoneNo, String newPhoneNo){
+        this.oldPhone = oldPhoneNo;
+        this.newPhone = newPhoneNo;
+    }
+    @Override
+    public CommandResult execute(){
+        //PhoneList pList = addressBook.
+        UniquePersonList people = addressBook.getAllPersons();
+        ReadOnlyPerson toBeRemoved = null;
+        //to loop thru every single person in the address book
+        for(ReadOnlyPerson person: people.immutableListView()){
+            //test to see if phone number exists in the address book
```

Comment:
Remember the discussion on comments in the Lecture? 😉 Going forward don't state the obvious.